### PR TITLE
TextEditing inverted coordinates fix.

### PR DESCRIPTION
**Pull request by jojolepro**
^ :D

blame jojo >_<

### CODE REVIEW

File: amethyst_ui/src/text.rs [hunk 1]
```diff
@@ -372,17 +373,18 @@ impl<'a> System<'a> for UiKeyboardSystem {
...
-                                .fold(INFINITY, |lowest, &(_, t)| lowest.min(t.global_z));
+                                .fold(-INFINITY, |highest, &(_, t)| highest.max(t.global_z));
                             // Then filter by it
-                            eligible.retain(|&(_, t)| t.global_z == lowest_z);
+                            eligible.retain(|&(_, t)| t.global_z == highest_z);
                             // We may still have ambiguity as to what to select at this point,
                             // so we'll resolve that by selecting the most recently created
                             // element.
                             focused.entity = eligible.iter().fold(None, |most_recent, &(e, _)| {
+                                println!("eligible {:?}!", e);
```
Comment (Xaeroxe) [inline amethyst_ui/src/text.rs]: Woops

File: amethyst_ui/src/text.rs [hunk 2]
```diff
@@ -372,17 +373,18 @@ impl<'a> System<'a> for UiKeyboardSystem {
...
                             // In instances of ambiguity we want to select the element with the
-                            // lowest Z order, so we need to find the lowest Z order value among
-                            // eligible elements
-                            let lowest_z = eligible
+                            // highest Z order, so we need to find the highest Z order value among
+                            // eligible elements.
+                            let highest_z = eligible
                                 .iter()
-                                .fold(INFINITY, |lowest, &(_, t)| lowest.min(t.global_z));
+                                .fold(-INFINITY, |highest, &(_, t)| highest.max(t.global_z));
```
Comment (azriel91) [inline amethyst_ui/src/text.rs]: can you use [`NEG_INFINITY`](https://doc.rust-lang.org/std/f32/constant.NEG_INFINITY.html) instead?

File: amethyst_ui/src/text.rs (hunk 1 above)
Comment (jojolepro) [reply amethyst_ui/src/text.rs]: thanks

File: amethyst_ui/src/text.rs (hunk 2 above)
Comment (jojolepro) [reply amethyst_ui/src/text.rs]: done

**Comment by jojolepro** [comment]
merge manually please

**Pull request merged**